.github/workflows/codecov.yml

```
@@ -21,7 +21,7 @@ jobs:
           npm install
           npm run bootstrap
           npm run build:all
-          npm run test:jest
+          npm run test:codecov
```

with the suggested approach on test:jest command you can simply pass `4` as value of maxWorker and in case you dont pass it it will use the fallback (default ) value which is 8 .
```suggestion
          npm run test:jest 4
```